### DISCUSSION TITLE
Show why a video is risking deletion

### DISCUSSION THREAD

**Discussion by Garbee** [MrBrax/LiveStreamDVR#251] [Ideas]
I see some videos have a trash icon that says "Risking Deletion". It would be useful to know why they are in this state. I have thousands of GB more storage allocated and hundreds more VoDs configured to keep. I don't understand why anything is currently in this state. So adding something to the interface to show what logic is triggering this would be useful. To help fix the configuration before things get deleted.

**Comment by MrBrax** [comment]
i'll add something. it just means that the streamer is not affiliated with twitch and it's highly likely that the vod will be deleted within two days (14 total)

**Reply by Garbee** [reply]
Ooh, so it is risking deletion on *twitch* not locally. That is slightly confusing then.

Perhaps be more specific with "Risking Deletion on Twitch"?
